### PR TITLE
Enable docker-py compat. testing w/ ignored result

### PR DESCRIPTION
**Pull request by cevich**
Partially address #9229 

Significant bitrot results in almost immediate test failure.  This
commit adds only the very basic, bare-minimum needed to get them
started directly on the host.

***TESTING RESULTS ARE IGNORED***

### CODE REVIEW

File: Makefile
Comment (jwhonce) [inline Makefile:117]: Comment not valid. Empty value will search whole repository for tests...

**Review by jwhonce — CHANGES_REQUESTED** [review]

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming @jwhonce 's comment is addressed.

File: Makefile
Comment (cevich) [reply Makefile:117]: oof, woops.  too quick to dismiss.  I see what you're saying now.

Comment (cevich) [reply Makefile]: okay, I just removed the thing.  Simpler that way.

File: contrib/cirrus/setup_environment.sh
Comment (jwhonce) [inline contrib/cirrus/setup_environment.sh:205]:
```suggestion
        pip install pytest requests-mock
```

**Comment by edsantiago** [comment]
Eek. This is beyond my ability to review, it's more pythony than what I know. If it passes CI -- and it seems to have -- it LGTM.

I think the original issue was asking for a way to run these tests from the command line, though? Is there a way to do that? Do we even want to?

**Comment by cevich** [comment]
> I think the original issue was asking for a way to run these tests from the command line, though? Is there a way to do that? Do we even want to?

Eventually, this would be nice for any developers wanting to work on it away from CI.  Jhon and I chatted and we think this can be left up to a future PR.

**Comment by cevich** [comment]
> I think the original issue was

In the interest of transparency, I'm not expecting merge of this PR to close #9229.   A large part of getting this task working was about environment setup.  So my intention is to open a new PR to handle the "running from Makefile" issue, now that the needed environment is a known-quantity.

**Pull request merged**
